src/http/routing.rs

```diff
@@ -983,16 +1131,42 @@ impl<'a> RouteInfo<'a> {
                 Route::GuildsIdEmojis(guild_id),
                 Cow::from(Route::guild_emojis(guild_id)),
             ),
+            RouteInfo::CreateFollowupMessage {
+                application_id,
+                interaction_token,
+                wait
```

Comment:
```suggestion
                wait,
```
Missing trailing comma.